doc/release-notes.md

```diff
@@ -99,6 +99,10 @@ Wallet
 - RPC `getreceivedbylabel` now returns an error, "Label not found
   in wallet" (-4), if the label is not in the address book. (#25122)
 
+- RPC `listunspent` now has a new argument `include_immature_coinbase`
+  to include coinbase UTXOs that don't met the minimum spendability 
```

Comment:
nit 
```suggestion
  to include coinbase UTXOs that don't meet the minimum spendability 
```

Reply:
should hire you as translator, pushed 👍🏼.